src/main/java/org/mockbukkit/mockbukkit/plugin/PluginManagerMock.java

```diff
@@ -407,6 +407,25 @@ public void registerLoadedPlugin(@NotNull Plugin plugin)
 		plugin.onLoad();
 	}
 
+	/**
+	 * Load a plugin from a class. The description will be guessed as best as possible.
+	 *
+	 * @param class1 The plugin to load.
+	 * @return The loaded plugin.
+	 */
+	public @NotNull JavaPlugin loadPlugin(@NotNull Class<? extends JavaPlugin> class1)
+	{
+		try
+		{
+			return loadPlugin(class1, new Object[0]);
+		}
+		catch (PluginLoadException ignored)
+		{
+			PluginDescriptionFile description = new PluginDescriptionFile(class1.getSimpleName(), "0.0.0", class1.getSimpleName());
```

Comment:
According to the Javadocs it has to be the full location: https://jd.papermc.io/paper/1.21.7/org/bukkit/plugin/PluginDescriptionFile.html#%3Cinit%3E(java.lang.String,java.lang.String,java.lang.String)
```suggestion
			PluginDescriptionFile description = new PluginDescriptionFile(class1.getSimpleName(), "0.0.0", class1.getName());
```